src/Pact/Native/Ops.hs

```diff
@@ -154,16 +154,22 @@ powImpl i as@[TLiteral a _,TLiteral b _] = do
     | otherwise = twoArgIntOpGas x x *> odds (x * x) (y `quot` 2) (x * z)
 powImpl i as = argsError i as
 
+getThreshold :: Eval e IntThreshold
+getThreshold = 
+  ifExecutionFlagSet' FlagDisablePact47 ThresholdPrePact47 ThresholdPact47
```

Comment:
Can you please add a comment here to describe how exactly these flags have changed the threshold?

Reply:
Sure.